packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -81,6 +81,9 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
   /* @conditional-compile-remove(one-to-n-calling) */
   const { callStatus } = useSelector(callStatusSelector);
 
+  /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+  const disableControlsForHoldScreen = adapter.getState().page === 'hold';
```

Comment:
can this be done at the selector level instead of plumbing through the composite? i.e. the usePropsFor(CameraButton) will return a `disabled: true` when on hold

Reply:
(note: if we need different strings for tooltips while being on hold that could be tough)

Reply:
will we want to add it at the selector level for when Contoso makes a component app? doing it here will allow Contoso to still use the selectors and as well give them the choice of what their behavior would be.

I will talk to Alex P about the strings, I think we should be able to set the strings dynamically in a similar fashion to the announcer component and how we have used that before.

Reply:
Hmm good question for parking lot, the reason I argue for it is the composite should have less internal logic of its own and just display whatever the state gives us, but you're right that it might be hard for contoso to understand why a button is disabled. Otherwise perhaps this should live in a composite-level selector

Reply:
Do we have an existing component being hidden by selectors?

Reply:
I do like the idea of it being in a composite level selector, get the best of both worlds. Is there a situation we already do this?

Reply:
> Do we have an existing component being disabled by selectors?
Yeah the camera and mic buttons depending on certain state from the calling sdk:
![image](https://user-images.githubusercontent.com/2684369/185200584-2ffaeb9e-ce44-4779-886a-754466e8aad2.png)

> I do like the idea of it being in a composite level selector, get the best of both worlds. Is there a situation we already do this?
Yeah, check out the selectors folder in the the individual composites dir and there should be some examples of applying composite-level selectors